Doc builds broken

### What happened?

#408 broke the doc builds.

### What command did you use?

```shell
make -C docs/ SPHINXOPTS="-W" BUILDDIR="$HOME/docs" OUTDIR="${CURBRANCH:-html}" html
```


### What version of the software are you running?

master

### How are you running this software?

Other

### Is your data BIDS valid?

Yes

### Are you reusing any previously computed results?

No

### Please copy and paste any relevant log output.

```shell
make: Entering directory '/home/runner/work/sdcflows/sdcflows/docs'
PYTHONPATH=/home/runner/work/sdcflows/sdcflows SPHINX_APIDOC_OPTIONS=members,show-inheritance sphinx-build -b html -d /home/runner/docs/doctrees  -W . /home/runner/docs/master
Running Sphinx v7.2.6
making output directory... done
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
loading intersphinx inventory from https://numpy.org/doc/stable/objects.inv...
loading intersphinx inventory from https://docs.scipy.org/doc/scipy/objects.inv...
loading intersphinx inventory from https://matplotlib.org/stable/objects.inv...
loading intersphinx inventory from https://bids-standard.github.io/pybids/objects.inv...
loading intersphinx inventory from https://nipy.org/nibabel/objects.inv...
loading intersphinx inventory from https://nipype.readthedocs.io/en/latest/objects.inv...
loading intersphinx inventory from https://www.nipreps.org/niworkflows/objects.inv...
loading intersphinx inventory from https://www.nipreps.org/smriprep/objects.inv...
loading intersphinx inventory from https://www.templateflow.org/python-client/objects.inv...
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.config.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.fieldmaps.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.transform.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.cli.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.cli.main.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.cli.parser.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.cli.workflow.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.data.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.interfaces.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.interfaces.brainmask.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.interfaces.bspline.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.interfaces.epi.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.interfaces.fmap.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.interfaces.reportlets.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.interfaces.utils.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.utils.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.utils.bimap.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.utils.epimanip.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.utils.misc.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.utils.phasemanip.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.utils.telemetry.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.utils.tools.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.utils.wrangler.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.viz.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.viz.utils.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.workflows.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.workflows.ancillary.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.workflows.base.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.workflows.outputs.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.workflows.apply.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.workflows.apply.correction.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.workflows.apply.registration.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.workflows.fit.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.workflows.fit.base.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.workflows.fit.fieldmap.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.workflows.fit.pepolar.rst.
Creating file /home/runner/work/sdcflows/sdcflows/docs/api/sdcflows.workflows.fit.syn.rst.
building [mo]: targets for 0 po files that are out of date
writing output... 
building [html]: targets for 9 source files that are out of date
updating environment: [new config] 46 added, 0 changed, 0 removed
reading sources... [  2%] api
reading sources... [  4%] api/sdcflows.cli
reading sources... [  7%] api/sdcflows.cli.main
reading sources... [  9%] api/sdcflows.cli.parser
reading sources... [ 11%] api/sdcflows.cli.workflow
reading sources... [ 13%] api/sdcflows.config
reading sources... [ 15%] api/sdcflows.data
reading sources... [ 17%] api/sdcflows.fieldmaps
reading sources... [ 20%] api/sdcflows.interfaces
reading sources... [ 22%] api/sdcflows.interfaces.brainmask
reading sources... [ 24%] api/sdcflows.interfaces.bspline
reading sources... [ 26%] api/sdcflows.interfaces.epi
reading sources... [ 28%] api/sdcflows.interfaces.fmap
reading sources... [ 30%] api/sdcflows.interfaces.reportlets
reading sources... [ 33%] api/sdcflows.interfaces.utils
reading sources... [ 35%] api/sdcflows.transform
reading sources... [ 37%] api/sdcflows.utils
reading sources... [ 39%] api/sdcflows.utils.bimap
reading sources... [ 41%] api/sdcflows.utils.epimanip
reading sources... [ 43%] api/sdcflows.utils.misc
reading sources... [ 46%] api/sdcflows.utils.phasemanip
reading sources... [ 48%] api/sdcflows.utils.telemetry
reading sources... [ 50%] api/sdcflows.utils.tools
reading sources... [ 52%] api/sdcflows.utils.wrangler
reading sources... [ 54%] api/sdcflows.viz
reading sources... [ 57%] api/sdcflows.viz.utils
reading sources... [ 59%] api/sdcflows.workflows
reading sources... [ 61%] api/sdcflows.workflows.ancillary
reading sources... [ 63%] api/sdcflows.workflows.apply
reading sources... [ 65%] api/sdcflows.workflows.apply.correction
231214-14:25:09,919 nipype.workflow INFO:
	 Generated workflow graph: /home/runner/docs/wf_directive/api/unwarp_wf/sdcflows-workflows-apply-correction-1.png (graph2use=orig, simple_form=True).
Downloading https://templateflow.s3.amazonaws.com/tpl-MNI152NLin2009cAsym/tpl-MNI152NLin2009cAsym_res-02_desc-brain_mask.nii.gz

  0%|          | 0.00/29.6k [00:00<?, ?B/s]
100%|██████████| 29.6k/29.6k [00:00<00:00, 10.9MB/s]
Downloading https://templateflow.s3.amazonaws.com/tpl-MNI152NLin2009cAsym/tpl-MNI152NLin2009cAsym_res-01_label-brain_probseg.nii.gz

  0%|          | 0.00/3.93M [00:00<?, ?B/s]
 56%|█████▌    | 2.19M/3.93M [00:00<00:00, 11.6MB/s]
100%|██████████| 3.93M/3.93M [00:00<00:00, 18.6MB/s]
231214-14:25:16,659 nipype.interface WARNING:
	 FSLOUTPUTTYPE environment variable is not set. Setting FSLOUTPUTTYPE=NIFTI
231214-14:25:16,661 nipype.interface WARNING:
	 FSLOUTPUTTYPE environment variable is not set. Setting FSLOUTPUTTYPE=NIFTI
231214-14:25:17,209 nipype.workflow INFO:
	 Generated workflow graph: /home/runner/docs/wf_directive/api/pepolar_estimate_wf/sdcflows-workflows-fit-pepolar-1.png (graph2use=orig, simple_form=True).
231214-14:25:17,445 nipype.workflow INFO:
	 Generated workflow graph: /home/runner/docs/wf_directive/api/pepolar_estimate_wf/sdcflows-workflows-fit-pepolar-1.svg (graph2use=orig, simple_form=True).
231214-14:25:17,712 nipype.workflow INFO:
	 Generated workflow graph: /home/runner/docs/wf_directive/api/pepolar_estimate_wf/sdcflows-workflows-fit-pepolar-1.pdf (graph2use=orig, simple_form=True).
231214-14:25:17,739 nipype.interface WARNING:
	 FSLOUTPUTTYPE environment variable is not set. Setting FSLOUTPUTTYPE=NIFTI
231214-14:25:17,742 nipype.interface WARNING:
	 FSLOUTPUTTYPE environment variable is not set. Setting FSLOUTPUTTYPE=NIFTI
231214-14:25:18,371 nipype.workflow INFO:
	 Generated workflow graph: /home/runner/docs/wf_directive/api/pepolar_estimate_wf/sdcflows-workflows-fit-pepolar-2.png (graph2use=orig, simple_form=True).
231214-14:25:18,621 nipype.workflow INFO:
	 Generated workflow graph: /home/runner/docs/wf_directive/api/pepolar_estimate_wf/sdcflows-workflows-fit-pepolar-2.svg (graph2use=orig, simple_form=True).
231214-14:25:18,890 nipype.workflow INFO:
	 Generated workflow graph: /home/runner/docs/wf_directive/api/pepolar_estimate_wf/sdcflows-workflows-fit-pepolar-2.pdf (graph2use=orig, simple_form=True).
reading sources... [ 80%] api/sdcflows.workflows.fit.syn
231214-14:25:19,639 nipype.workflow INFO:
	 Generated workflow graph: /home/runner/docs/wf_directive/api/syn_sdc_wf/sdcflows-workflows-fit-syn-1.png (graph2use=orig, simple_form=True).
231214-14:25:19,893 nipype.workflow INFO:
	 Generated workflow graph: /home/runner/docs/wf_directive/api/syn_sdc_wf/sdcflows-workflows-fit-syn-1.svg (graph2use=orig, simple_form=True).
231214-14:25:20,161 nipype.workflow INFO:
	 Generated workflow graph: /home/runner/docs/wf_directive/api/syn_sdc_wf/sdcflows-workflows-fit-syn-1.pdf (graph2use=orig, simple_form=True).
231214-14:25:20,848 nipype.workflow INFO:
	 Generated workflow graph: /home/runner/docs/wf_directive/api/syn_sdc_wf/sdcflows-workflows-fit-syn-2.png (graph2use=orig, simple_form=True).
231214-14:25:21,109 nipype.workflow INFO:
	 Generated workflow graph: /home/runner/docs/wf_directive/api/syn_sdc_wf/sdcflows-workflows-fit-syn-2.svg (graph2use=orig, simple_form=True).
231214-14:25:21,388 nipype.workflow INFO:
	 Generated workflow graph: /home/runner/docs/wf_directive/api/syn_sdc_wf/sdcflows-workflows-fit-syn-2.pdf (graph2use=orig, simple_form=True).
reading sources... [ 83%] api/sdcflows.workflows.outputs
reading sources... [ 85%] changes
reading sources... [ 87%] cli
reading sources... [ 89%] examples
reading sources... [ 91%] index
reading sources... [ 93%] installation
reading sources... [ 96%] links
reading sources... [ 98%] methods
0.00s - Debugger warning: It seems that frozen modules are being used, which may
0.00s - make the debugger miss breakpoints. Please pass -Xfrozen_modules=off
0.00s - to python to disable frozen modules.
0.00s - Note: Debugging will proceed. Set PYDEVD_DISABLE_FILE_VALIDATION=1 to disable this validation.
0.00s - Debugger warning: It seems that frozen modules are being used, which may
0.00s - make the debugger miss breakpoints. Please pass -Xfrozen_modules=off
0.00s - to python to disable frozen modules.
0.00s - Note: Debugging will proceed. Set PYDEVD_DISABLE_FILE_VALIDATION=1 to disable this validation.
reading sources... [100%] notebooks/SDC - Theory and physics

Warning, treated as error:
autodoc: failed to import module 'parser' from module 'sdcflows.cli'; the following exception was raised:
sdcflows
make: *** [Makefile:61: html] Error 2
make: Leaving directory '/home/runner/work/sdcflows/sdcflows/docs'
Error: Process completed with exit code 2.
```


### Additional information / screenshots

Example CI failure: https://github.com/nipreps/sdcflows/actions/runs/7210198586/job/19642804807

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -370,7 +370,7 @@ jobs:
           name: Install Graphviz & pandoc
           command: |
             sudo apt-get update -y
-            sudo apt-get install -y --no-install-recommends graphviz pandoc
+            sudo apt-get install -y --no-install-recommends graphviz pandoc texlive
       - run:
           name: Install deps
           command: |
@@ -384,7 +384,7 @@ jobs:
             source /tmp/venv/bin/activate
             python -m hatch version | tail -n1 | xargs
             BRANCH=$( echo $CIRCLE_BRANCH | sed 's+/+_+g' )
-            make -C docs SPHINXOPTS="-W" BUILDDIR="$HOME/docs" OUTDIR=${CIRCLE_TAG:-$BRANCH} html
+            make -C docs SPHINXOPTS="-W -v" BUILDDIR="$HOME/docs" OUTDIR=${CIRCLE_TAG:-$BRANCH} html
       - store_artifacts:
           path: ~/docs/
 

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -384,6 +384,7 @@ jobs:
             source /tmp/venv/bin/activate
             python -m hatch version | tail -n1 | xargs
             BRANCH=$( echo $CIRCLE_BRANCH | sed 's+/+_+g' )
+            python -c "from templateflow.api import get; get('MNI152NLin2009cAsym', desc='brain', resolution=1, suffix='T1w')"
             make -C docs SPHINXOPTS="-W -v" BUILDDIR="$HOME/docs" OUTDIR=${CIRCLE_TAG:-$BRANCH} html
       - store_artifacts:
           path: ~/docs/

--- a/docs/notebooks/SDC - Theory and physics.ipynb
+++ b/docs/notebooks/SDC - Theory and physics.ipynb
@@ -142,7 +142,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "data = np.asanyarray(nb.load(get(\"MNI152NLin2009cAsym\", desc=\"brain\", resolution=1, suffix=\"T1w\")).dataobj)"
+    "data = np.asanyarray(nb.load(get(\"MNI152NLin2009cAsym\", desc=\"brain\", resolution=1, suffix=\"T1w\")).dataobj);"
    ]
   },
   {

